src/controllers/dynakube/activegate/capability/capability.go

```diff
@@ -100,8 +100,9 @@ func (c *capabilityBase) ShouldCreateService() bool {
 
 // Note:
 // Caller must set following fields:
-//   Image:
-//   Resources:
+//
+//	Image:
+//	Resources:
```

Comment:
Is that comment as a whole even relevant anymore?

Reply:
Yes, it is.

Reply:
It's relevant but doesn't satisfy requirements for docstrings and potential problem it describes would be immediately caught. Therefore it's now removed, thanks.